examples/conversationbot.py

```diff
@@ -96,23 +96,23 @@ def skip_location(bot, update):
 
 def bio(bot, update):
     user = update.message.from_user
-    logger.info("Bio of %s: %s" % (user.first_name, update.message.text))
+    logger.info("Bio of %s: %s", user.first_name, update.message.text)
     update.message.reply_text('Thank you! I hope we can talk again some day.')
 
     return ConversationHandler.END
 
 
 def cancel(bot, update):
     user = update.message.from_user
-    logger.info("User %s canceled the conversation." % user.first_name)
+    logger.info("User %s canceled the conversation.", user.first_name)
     update.message.reply_text('Bye! I hope we can talk again some day.',
                               reply_markup=ReplyKeyboardRemove())
 
     return ConversationHandler.END
 
 
 def error(bot, update, error):
-    logger.warn('Update "%s" caused error "%s"' % (update, error))
+    logger.warn('Update "%s" caused error "%s"', update, error)
```

Comment:
Can you change this to `logger.warning`? `Logger.warn` is deprecated.
https://docs.python.org/3/library/logging.html#logging.Logger.warning